tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -574,17 +574,19 @@ def test_assigned_to_pagination(self, patched_params_update):
             ag.update(status=GroupStatus.RESOLVED, resolved_at=before_now(seconds=5))
             GroupAssignee.objects.assign(ag, self.user)
 
-        patched_params_update.side_effect = [
-            (self.organization.id, {"project": [self.project.id]}),
-            (self.organization.id, {"project": [self.project.id]}),
-            (self.organization.id, {"project": [self.project.id]}),
-            (self.organization.id, {"project": [self.project.id]}),
-            (self.organization.id, {"project": [self.project.id], "orderby": ["-last_seen"]}),
-            (self.organization.id, {"project": [self.project.id]}),
-            (self.organization.id, {"project": [self.project.id]}),
-            (self.organization.id, {"project": [self.project.id]}),
-            (self.organization.id, {"project": [self.project.id]}),
-        ]
+        def _my_patched_params(query_params, **kwargs):
```

Comment:
```suggestion
        # This side_effect is meant to override the `calculate_hits` snuba query specifically. 
        # If this test is failing it's because the -last_seen override is being applied to a different snuba query.
        def _my_patched_params(query_params, **kwargs):
```